packages/amesos2/src/KLU2/Source/CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ klu2_memory.hpp
   )
 
 #
-# C) Define the targets for package's example
+# C) Define the targets for package's test
 #
 
-TRIBITS_ADD_EXAMPLE_DIRECTORIES(examples)
+TRIBITS_ADD_TEST_DIRECTORIES(tests)
```

Comment:
Note for the reviewer(s): I promoted the example to being a "standalone test" of the `KLU2` sources.